Dockerfile-base-cpu

```diff
@@ -59,6 +59,7 @@ ENV OMPI_ALLOW_RUN_AS_ROOT_CONFIRM ${WITH_MPI:+1}
 # installed.
 RUN pip install determined && pip uninstall -y determined
 
+RUN pip install -r /tmp/det_dockerfile_scripts/requirements.txt
```

Comment:
We don't ever publish `base` images directly. They're created as a base for our `default` images. This requirement doesn't have to be in both Dockerfiles.